src/Mod/TechDraw/TechDrawTools/CommandMoveView.py

```diff
@@ -0,0 +1,98 @@
+# *   for detail see the LICENCE text file.                                 *
+# *                                                                         *
+# *   This program is distributed in the hope that it will be useful,       *
+# *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+# *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
+# *   GNU Library General Public License for more details.                  *
+# *                                                                         *
+# *   You should have received a copy of the GNU Library General Public     *
+# *   License along with this program; if not, write to the Free Software   *
+# *   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  *
+# *   USA                                                                   *
+# *                                                                         *
+# ***************************************************************************
+"""Provides the TechDraw MoveView GuiCommand."""
+
+__title__ = "TechDrawTools.CommandMoveView"
+__author__ = "WandererFan"
+__url__ = "http://www.freecadweb.org/index.html"
+__version__ = "00.01"
+__date__ = "2022/01/11"
+
+from PySide.QtCore import QT_TRANSLATE_NOOP
+from PySide import QtCore
+import PySide.QtGui as QtGui
+
+import FreeCAD as App
+import FreeCADGui as Gui
+import TechDraw
+from TechDrawTools import TDToolsUtil
+from TechDrawTools import TDToolsMovers
+from TechDrawTools import TaskMoveView
+import os
+
+class CommandMoveView:
+    """Moves a View from current Page to a different Page."""
+
+    def __init__(self):
+        """Initialize variables for the command that must exist at all times."""
+        pass
+
+    def GetResources(self):
+        """Return a dictionary with data that will be used by the button or menu item."""
+        return {'Pixmap': 'actions/TechDraw_MoveView.svg',
+                'Accel': "",
+                'MenuText': QT_TRANSLATE_NOOP("MoveView", "Move View"),
+                'ToolTip': QT_TRANSLATE_NOOP("MoveView", "Move a View to a new Page")}
+
+    def Activated(self):
+        """Run the following code when the command is activated (button press)."""
+#        print("Activated()")
+        sel = Gui.Selection.getSelection()
+
+        vName = ""
+        views = list()
+        for o in sel:
+            if o.isDerivedFrom("TechDraw::DrawView"):
+                views.append(o)
+        if views:
+            v = views[0]
+            vName = v.Name
+
+        tpName = ""
+        fpName = ""
+        pages = list()
+        for o in sel:
+            if o.isDerivedFrom("TechDraw::DrawPage"):
+                pages.append(o)
+        if pages:
+            fromPage = pages[0]
+            fpName = fromPage.Name
+        if len(pages) > 1:
+            toPage = pages[1]
+            tpName = toPage.Name
+
+        self.ui  = TaskMoveView.TaskMoveView()
```

Comment:
This is the same code as in CommandCopyView.py. In the future, it could be handy to move it to a module with a function returning vname, fpname and tpname?